### ISSUE TITLE
[Feature Request]:  Bayesian Mediation

### ISSUE THREAD

**Issue by TarandeepKang** [jasp-stats/jasp-issues#1630]
### Use-case

Likely whenever frequentist mediation would be used, but with the inherent advantages of Bayesian analysis

### Is your feature request related to a problem?

Not a problem per se, just the inclusion of a method that the team has already developed

### Describe the solution you would like

The inclusion of the Bayesian counterpart to Frequentist mediation perhaps as in: Nuijten, M.B., Wetzels, R., Matzke, D. et al. A default Bayesian hypothesis test for mediation. Behav Res 47, 85–97 (2015). https://doi.org/10.3758/s13428-014-0470-2

### Describe alternatives that you have considered

Going straight to using the R package, but it does not seem that this can be easily downloaded.

### Additional context

Thank you for implementing frequentist mediation analysis, using the GUI, and for the flexibility of structural equation modelling for more complicated analysis. I recently came across this Bayesian implementation, and it seems the logical counterpart to your frequent test implementation, and would follow nicely with the ethos of making Bayesian analysis accessible. Not least, because it seems like the package can no longer be downloaded from CRAN?

**Comment by juliuspf** [comment]
Hi @TarandeepKang, thank you for the request. I will ask around :)

**Comment by juliuspf** [comment]
@EJWagenmakers, what do you think?

**Comment by EJWagenmakers** [comment]
Yes, would be good. However, the package is no longer on CRAN, and I would want to give this another think (and see how this can be generalized)

**Comment by TarandeepKang** [comment]
I can help you there, I'm afraid, Thomas, EJ is the author on the paper, so I guess it's his baby (well at least partly :-))?
